code/modules/power/supermatter/supermatter.dm

```diff
@@ -667,11 +667,11 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 		if(gas_comp[/datum/gas/carbon_dioxide] && gas_comp[/datum/gas/oxygen])
 			var/carbon_dioxide_pp = env.return_pressure() * gas_comp[/datum/gas/carbon_dioxide]
 			var/consumed_carbon_dioxide = clamp(((carbon_dioxide_pp - CO2_CONSUMPTION_PP) / (carbon_dioxide_pp + CO2_PRESSURE_SCALING)), CO2_CONSUMPTION_RATIO_MIN, CO2_CONSUMPTION_RATIO_MAX)
-			consumed_carbon_dioxide = min(consumed_carbon_dioxide * gas_comp[/datum/gas/carbon_dioxide] * combined_gas, removed.gases[/datum/gas/carbon_dioxide][MOLES], removed.gases[/datum/gas/oxygen][MOLES] * INVERSE(0.5))
+			consumed_carbon_dioxide = min(consumed_carbon_dioxide * gas_comp[/datum/gas/carbon_dioxide] * combined_gas, removed.gases[/datum/gas/carbon_dioxide][MOLES] * INVERSE(0.5), removed.gases[/datum/gas/oxygen][MOLES] * INVERSE(0.5))
 			if(consumed_carbon_dioxide)
-				removed.gases[/datum/gas/carbon_dioxide][MOLES] -= consumed_carbon_dioxide
+				removed.gases[/datum/gas/carbon_dioxide][MOLES] -= consumed_carbon_dioxide * 0.5
 				removed.gases[/datum/gas/oxygen][MOLES] -= consumed_carbon_dioxide * 0.5
-				removed.gases[/datum/gas/pluoxium][MOLES] += consumed_carbon_dioxide * 0.5
+				removed.gases[/datum/gas/pluoxium][MOLES] += consumed_carbon_dioxide * 0.05
```

Comment:
is it really 10x as producing as it should be?

Reply:
more or less, before people were able to run a CO2 setup and have all the pluox removed from the chamber, now they can't because the buildup is much higher than the removal. Maybe 10x reduction might be harsh but with the higher generation of it you're able to produce decent amounts regardless. If you want i can reduce it to 6/7 times less if sounds better to you

Reply:
changed this to 5x reduction instead of 10x to prepare for a next PR to add a proper way to increase the pluox production